### ISSUE TITLE
As a user

### ISSUE THREAD

**Issue by traceWater** [traceWater/social-media-app#2]

**Comment by traceWater** [comment]
Complexity 2
Time Box     3-hrs

User expectation: See posted data


As a developer, I have to know how to create a Post Form with components to hold state for lists of posts and lists of users, they are separate and the post has the user id of who posts?

**Comment by luceroweb** [comment]
@traceWater yes, I think you're right that you'll need to have a separate data object for users. I'll write up a separate ticket for that now. In the mean time, you can use the user names "traceWater" and "luceroweb", just like I did in #10 .
